service-tests/xml.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+const colorscheme = require('../lib/colorscheme.json');
+const mapValues = require('lodash.mapvalues');
+
+const colorsB = mapValues(colorscheme, 'colorB');
+
+const t = new ServiceTester({ id: 'badge/dynamic/xml', title: 'User Defined XML Source Data' });
```

Comment:
Can you add a test for querying an attribute (`/addon/@id`)?

Reply:
Knew i forgot something! Glad you caught this.